packages/runtime-dom/src/patchProp.ts

```diff
@@ -122,6 +122,11 @@ function shouldSetAsProp(
     return false
   }
 
+  // #12211 <a text> must be set as attribute
+  if (key === 'text' && el.tagName === 'A') {
+    return false
+  }
```

Comment:
`<a>` element doesn't have a `text` attribute while `HTMLAnchorElement` has a `text` property. Not sure if we should add this or should let users to use `^text="..."` instead.

Reply:
I missed this detail, thanks for pointing it out.

Reply:
I'm not saying we shouldn't add this 😅

I just want to first discuss the criteria we use to handle such situations and whether we should first check if other frameworks have any special handling. I think we can keep this open for now.

Reply:
It seems almost all major frameworks directly renders `text` as an attribute. So I suppose it's fine for us to align with this behavior?